pkg/gardenlet/controller/shoot/shoot/reconciler.go

```diff
@@ -283,8 +283,7 @@ func (r *Reconciler) prepareOperation(ctx context.Context, log logr.Logger, shoo
 			return nil, reconcile.Result{}, utilerrors.WithSuppressed(syncErr, updateErr)
 		}
 
-		log.Info("Shoot is failed or ignored, do not start operation")
-		return nil, reconcile.Result{}, nil
+		return nil, r.scheduleNextSync(log, shoot, false, "Shoot is failed or ignored, reconciliation is not allowed"), nil
```

Comment:
So, do we want to requeue it even if it's failed?

Reply:
We do requeue failed and ignored shoots here. `scheduleNextSync` returns a `reconcile.Request` with `RequeueAfter` set.